### PR TITLE
feat(tile-converter): add transformation matrix generation for node

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js [hunk 1]
```diff
@@ -243,6 +243,35 @@ function convertNodes(
   }
 }
 
+/**
+ * Generate transformation matrix for node
```
Comment (ibgreen) [inline modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:247]: Can we add a bit more explanation?

File: modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js [hunk 2]
```diff
@@ -243,6 +243,35 @@ function convertNodes(
...
+ * @param node
+ * @param matrix
+ */
+function getCompositeTransformationMatrix(node, matrix) {
```
Comment (ibgreen) [inline modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:251]: Types please?

Comment (ibgreen) [inline modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:251]:
```suggestion
function getCompositeTransformationMatrix(node, targetMatrix: Matrix4 = new Matrix4()): Matrix4 {
```

**Review by ibgreen — APPROVED** [review]

File: modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:247]: Done

File: modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:252]: Will do in separate PR where we migrate this file from `js` -> `ts`

Comment (dryabinin-actionengine) [reply modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:252]: Will do in separate PR where we migrate this file from js -> ts

**Pull request merged**